src/libappimage/type1.c

```diff
@@ -152,3 +154,39 @@ appimage_handler appimage_type_1_create_handler() {
 
     return h;
 }
+
+uint match_type_1_magic_bites(const char* buffer) {
```

Comment:
`bytes`, not `bites`. And why does this even return a `uint`? The return value is obviously supposed to be `bool`.